### ISSUE TITLE
Any way to change background of pitch?

### ISSUE THREAD

**Issue by hax-ops** [haxball/haxball-issues#997]
I mean this gross png; https://prnt.sc/qweye3

Is there any way to make this solid color? 

I want this because on higher zooms like 2.5x, while moving this gross background gives your eyes a terrible experience. You focus on this gross instead of ball and player and it makes everything bad. So an option in video settings like; "Disable background" would be great.

**Comment by bartakabstina** [comment]
Yes you can, but you need to modify the map with any text editor. To do this, make the WIDTH=0 and HEIGHT=0, and add a new "sentence" called "color":"your_color_here"(your color has to be in hex, meaning you can't write red,blue,etc)
What I said might be confusing, so here's an example from a map:
`"bg":{"type":"grass","width":0,"height":0,"kickOffRadius":80,"cornerRadius":0,"color":"333184"}`
Here, as you can see the width and height of the map are zeros, and the color is 333184 which is something blueish. 

You can get the codes for colors on many HTML Color Codes Sites, but here are some of the most used ones:

white - FFFFFF
black - 000000
yellow - FFFF00
red - FF0000
blue - 0000FF
magenta - FF00F0
green - 00FF00
light blue- 00FFFF

**Comment by hax-ops** [comment]
I only play default maps. The feature I asked for was the default maps.

**Comment by bartakabstina** [comment]
Well then like edit the default map code

**Comment by hax-ops** [comment]
If I do that, I have to be admin and upload my map everytime I play. And this is not possible thing in league matches.

And also it effects everyone, the fix I wanted for is client side.

**Comment by Mirac35** [comment]
Then its only possible by injecting your own game-min.js to the game. So its still possible.

**Comment by xRoos3velt** [comment]
@spr1907 please add me on discord xRoos3velt#3441
